preview-src/index.ts

```diff
@@ -82,7 +82,7 @@ const onUpdateView = (() => {
 const updateImageSizes = throttle(() => {
   const imageInfo: { id: string, height: number, width: number }[] = []
   const images = document.getElementsByTagName('img')
-  if (images) {
+  if (images && images.length > 0) {
```

Comment:
Indirectly. When looking into the event listener, that clicking on the `xref:` (without any other changes) triggered a `cacheImageSizes` even though there are no images at all in the document I tested with.
It could alos be moved to a dedicated PR.

Reply:
Indeed, good catch